packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -1012,8 +1012,7 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
     if (copySampleCounts) {
       copy
         ..exclusiveSampleCount = exclusiveSampleCount
-        ..inclusiveSampleCount =
-            resetInclusiveSampleCount ? null : inclusiveSampleCount;
+        ..inclusiveSampleCount = inclusiveSampleCount;
```

Comment:
I swear I wanted to make this change before and we decided it wasn't correct... 😂 😭

Reply:
I think so too 😆  except this change needed to be made in combination with the change in `generateBottomUpRoots` to set `currentBottomUpRoot.inclusiveSampleCount = null` when `node.exclusiveSampleCount == 0`. After this PR, we should finally have this calculation right 😅